src/Arcus.Messaging.Tests.Integration/ServiceBus/ServiceBusClient.cs

```diff
@@ -33,62 +35,108 @@ public ServiceBusClient(KeyRotationConfig configuration, ITestOutputHelper logge
             _logger = logger;
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        /// <param name="entity"></param>
+        /// <returns></returns>
+        public async Task<AccessKeys> GetConnectionStringKeysAsync(ServiceBusEntity entity)
+        {
```

Comment:
Don't forget to guard this

Reply:
Removed entity parameter, so no guard needed anymore. Resolving.